packages/components/src/palette-edit/index.js

```diff
@@ -313,6 +313,13 @@ export default function PaletteEdit( {
 		! elements[ editingElement ].slug;
 	const elementsLength = elements.length;
 	const hasElements = elementsLength > 0;
+	const editPaletteItem = ( value, collection ) => {
+		const valueIndex = collection.findIndex(
+			( item ) => item.color === value
```

Comment:
Problem here: there could be two or more colors/gradients in the presets array with the same color/gradient value.

Also we should be checking for `item.gradient` for gradients, but that's beside the point 😄